services/src/pro/util/tests.rs

```diff
@@ -296,3 +310,87 @@ pub(in crate::pro) mod mock_oidc {
         result
     }
 }
+
+#[allow(clippy::missing_panics_doc)]
+pub async fn register_ndvi_workflow_helper(ctx: &ProInMemoryContext) -> (Workflow, WorkflowId) {
```

Comment:
Just as a thought:
Should we only compile this if(test)?

Maybe clippy would not bother then about the missing panics doc.
And we could not use it apart from test.

Reply:
Stackoverflow says that the Rust compiler doesn't include unused methods in the binary. So I don't think we need to do this. But we could.

Reply:
No, this is clear to me. It's just that clippy says, okay, this is a regular method, so why do you just panic?
And it is, we could use it, but we use it only in test. Wondering if we could just enclose it in #[cfg(test)] ?

Reply:
how about we make the whole file #cfg[test]?

Reply:
27586db60cfc6f20d0938060f84902b2f3d96ff1